src/model/webhook.rs

```diff
@@ -230,9 +285,14 @@ impl Webhook {
     ///
     /// # Errors
     ///
-    /// May return [`Error::Http`] if the content is illformed, or if the webhook's token is invalid.
-    /// Also, may return an [`Error::Json`] if there is an error deserializing Discord's response.
+    /// Returns an [`Error::Model`] if the [`token`] is `None`.
+    ///
+    /// May also return an [`Error::Http`] if the content is illformed, or if the webhook's token is invalid.
```

Comment:
```suggestion
    /// May also return an [`Error::Http`] if the content is malformed, or if the webhook's token is invalid.
```
As said, *malformed* is the common way to express this.